kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/subscription/DataBrokerSubscriberTest.kt

```diff
@@ -61,15 +63,13 @@ class DataBrokerSubscriberTest : BehaviorSpec({
                     databrokerTransporter.updateRandomFloatValue(vssPath)
 
                     then("The PropertyListener is notified about the change") {
-                        verify(timeout = 100L) {
+                        verify(timeout = 100L, exactly = 2) {
```

Comment:
shouldn't it be exactly 1?

Reply:
okay, I see

Reply:
Should've read the markdown file :)

Reply:
It's also part of the reason why we normall don't finish after verification but also take a look at the values received and check if we find the exact last updated value inside :D